src/main/java/net/pms/formats/v2/AudioAttribute.java

```diff
@@ -1,5 +1,5 @@
 /*
- * PS3 Media Server, for streaming any media to your PS3.
+ * PS3 Media Server, for streaming any medias to your PS3.
```

Comment:
@Sami32 ```media```, but we can't/shouldn't change another projects' header. The "rule" for these headers is that you do not change them, and I guess that should apply to spelling mistakes as well.